chrome/content/zotero/elements/itemDetails.js

```diff
@@ -283,6 +287,10 @@
 				}
 			}
 
+			// Wait to make sure the visibility and height are updated
+			let promise = new Promise(resolve => setTimeout(resolve, 0));
+			await promise;
```

Comment:
We can use `Zotero.Promise.delay(0)` here and below if we really need to do this, but is this actually reliable?